test/e2e/service.go

```diff
@@ -11,13 +11,15 @@ import (
 
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
nit: the header of this commit talks about unit tests, but these are not unit tests they are e2e tests

Reply:
done